File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/ticket_item/exception/TicketItemQuantityLargeException.java
package band.gosrock.domain.domains.ticket_item.exception;

import band.gosrock.common.exception.DuDoongCodeException;

public class TicketItemQuantityLargeException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new TicketItemQuantityLargeException();

    private TicketItemQuantityLargeException() {
        super(TicketItemErrorCode.TICKET_ITEM_QUANTITY_LARGER_THAN_SUPPLY_COUNT);}
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/ticket_item/exception/TicketItemQuantityLargeException.kt
package band.gosrock.domain.domains.ticket_item.exception

import band.gosrock.common.exception.DuDoongCodeException

class TicketItemQuantityLargeException private constructor() : DuDoongCodeException(TicketItemErrorCode.TICKET_ITEM_QUANTITY_LARGER_THAN_SUPPLY_COUNT) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = TicketItemQuantityLargeException()
    }
}
